File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task12_CLAUDE_claude-3-5-sonnet-20240620.java

import java.util.Scanner;

public class HighestScoringWord {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        scanner.close();

        String[] words = input.split(" ");
        String highestScoringWord = "";
        int highestScore = 0;

        for (String word : words) {
            int score = calculateScore(word);
            if (score > highestScore) {
                highestScore = score;
                highestScoringWord = word;
            }
        }

        System.out.println(highestScoringWord);
    }

    private static int calculateScore(String word) {
        int score = 0;
        for (char c : word.toCharArray()) {
            score += c - \'a\' + 1;
        }
        return score;
    }
}

File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task12_CLAUDE_claude-3-5-sonnet-20240620.cpp

#include <iostream>
#include <string>
#include <sstream>
#include <vector>

using namespace std;

int calculateScore(const string& word) {
    int score = 0;
    for (char c : word) {
        score += tolower(c) - \'a\' + 1;
    }
    return score;
}

int main() {
    string input;
    getline(cin, input);

    istringstream iss(input);
    vector<string> words{istream_iterator<string>{iss}, istream_iterator<string>{}};

    string highestScoringWord = "";
    int highestScore = 0;

    for (const string& word : words) {
        int score = calculateScore(word);
        if (score > highestScore) {
            highestScore = score;
            highestScoringWord = word;
        }
    }

    cout << highestScoringWord << endl;

    return 0;
}
